solr/core/src/test/org/apache/solr/cloud/MockSimpleZkController.java

```diff
@@ -26,14 +26,22 @@
 
 public class MockSimpleZkController extends ZkController {
 
+  /**
+   * @param cc Core container associated with this controller. cannot be null.
+   * @param zkServerAddress where to connect to the zk server
+   * @param zkClientConnectTimeout timeout in ms
+   * @param cloudConfig configuration for this controller. TODO: possibly redundant with
+   *     CoreContainer
+   * @param descriptorsSupplier a supplier of the current core descriptors. used to know which cores
+   */
```

Comment:
minor/subjective: seems to just repeat javadocs from super class, not sure if that's useful